src/views/Projects/index.jsx

```diff
@@ -2,26 +2,42 @@ import { hot } from 'react-hot-loader';
 import { Component, Fragment } from 'react';
 import { withStyles } from '@material-ui/core/styles';
 import Grid from '@material-ui/core/Grid';
-import TextField from '@material-ui/core/TextField';
 import Typography from '@material-ui/core/Typography';
+import AppBar from '@material-ui/core/AppBar';
 import projects from '../../data/loader';
 import ProjectCard from '../../components/ProjectCard';
+import SearchBox from '../../components/SearchBox';
 
 @hot(module)
 @withStyles(theme => ({
   container: {
-    paddingRight: 15,
-    paddingLeft: 15,
-    marginRight: 'auto',
-    marginLeft: 'auto',
+    paddingTop: 1 * theme.spacing.unit,
+    backgroundColor: theme.palette.primary.light,
+    minHeight: `calc(100vh - 180px - ${3 * theme.spacing.unit}px)`,
+    marginTop: `calc(180px + ${theme.spacing.unit}px)`,
```

Comment:
Replace `minHeight` and `marginTop` with:

```js
container: {
  ...
  height: '86vh',
  marginTop: `calc(14vh + ${3 * theme.spacing.unit}px)`,
}
```

_Note: To make the layout work, `height: '14vh',` needs to be added to the `header` class._

```js
header: {
  paddingTop: theme.spacing.unit,
  height: '14vh',
  background: 'linear-gradient(45deg, #364598, #6AE5F4 90%)',
},
```

Reply:
Also, we can add padding to container:

```js
container: {
  backgroundColor: theme.palette.primary.light,
  height: '86vh',
  marginTop: `calc(14vh + ${3 * theme.spacing.unit}px)`,
  padding: `0 ${2 * theme.spacing.unit}px`
}
```

We won't need the styling in `@media (min-width: 1024px)` anymore.

Reply:
I still get this 
<img width="1680" alt="screen shot 2018-06-21 at 4 41 20 am" src="https://user-images.githubusercontent.com/24544912/41683628-c0e53d32-750d-11e8-8f3c-cf64355ae6fa.png"> 
then I think it might be because of the SearchBox component, since it has marginTop, then i remove it and I get this, which is better since now the SearchBox is in the header
<img width="1680" alt="screen shot 2018-06-21 at 4 42 30 am" src="https://user-images.githubusercontent.com/24544912/41683735-08f804e2-750e-11e8-964f-47ce07c99b44.png">
but when I try on smaller screen, the SearchBox and second Typography aren't placed in the header
<img width="1680" alt="screen shot 2018-06-21 at 4 42 56 am" src="https://user-images.githubusercontent.com/24544912/41683806-424a7842-750e-11e8-95e7-c8e1e5b920eb.png">